includes/loader.php

```diff
@@ -14,13 +14,6 @@
 define( 'GOOGLESITEKIT_PLUGIN_BASENAME', plugin_basename( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
 define( 'GOOGLESITEKIT_PLUGIN_DIR_PATH', plugin_dir_path( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
 
-// Enable this ENV variable to load the new, refactored JavaScript code.
```

Comment:
This check removes the ability to toggle this feature on from the command-line though, right? Though come to think of it that won't work because PHP isn't reading our `ENV` from `npm run`, whoops.

Good call, but we'll want another way to enable this I guess. Where do we usually put feature flags like this?

Reply:
It could just be added to wp-config. If we end up creating automated tests for this, at some point, we could set that in Travis and define the constant in the e2e set up script or even in a e2e helper plugin. The value could still be sourced from an env variable, but you're right that it wouldn't get it from the `npm run` context.